src/client/scripts/createRoom.js

```diff
@@ -26,6 +32,19 @@ Template.createRoom.helpers({
 
     createRoomDisable: function() {
         return (Session.get("roomCodeAvailable") ? "" : "disabled");
+    },
+
+    getCategories: function() {
+        categoriesDep.depend();
+        return categories;
+    },
+
+    colorPicker: function(color) {
+        return {
+            id: "cardBackgroundColor",
+            type: "color",
```

Comment:
this is not supported by IE.  If this is a problem we can look for an alternative.  From what I could find all the alternatives were a pain in the ass to use/setup.  It is HTML5 so IE *might* support it in the future so it may never be an issue. I also dont think any1 at Zywave (or in the dev world in general) uses IE anyways.